<launch>
    <node pkg="turtlebot_transport_flexmansys" type="transport_flexmansys_main.py"
     name="transport_flexmansys_node_leonardo" output="screen">
    </node>
    <node pkg="turtlebot_transport_flexmansys" type="transport_flexmansys_send_goal.py"
     name="transport_flexmansys_node_server_leonardo" output="screen">
    </node>
    <node pkg="turtlebot_transport_flexmansys" type="transport_flexmansys_emergency_stop_node.py"
     name="transport_flexmansys_emergency_stop_node_leonardo" output="screen">
    </node>
    <node pkg="turtlebot_transport_flexmansys" type="transport_flexmansys_state_publisher.py"
     name="flexmansys_transport_node_leonardo_state" output="screen">
    </node>
</launch>